scripts/apps/search/components/actions-menu/Label.tsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {closeActionsMenu} from '../../helpers';
+import {gettext} from 'core/ui/components/utils';
 
-const Label: React.StatelessComponent<any> = (props) => {
-    const {gettextCatalog} = props.svc;
-
-    return React.createElement(
+const Label: React.StatelessComponent<any> = (props) =>
+    React.createElement(
         'li',
         null,
         React.createElement('div', {
             className: 'dropdown__menu-label',
-        }, gettextCatalog.getString(props.label),
+        }, gettext(props.label),
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.

Reply:
Here it is better to receive the translated label as proposed below.